Tests/TwoDimension/AreaTests.cs

```diff
@@ -156,6 +156,8 @@ public void PositionSet()
 			Assert.That(() => area.SetPosition(null, mockPosition.Object), Throws.ArgumentNullException);
 			Assert.That(() => area.SetPosition(mockLevel.Object, null), Throws.ArgumentNullException);
 
+			// TODO: Issue 13 (https://github.com/Wizcorp/TileSystem/issues/13)
```

Comment:
the issues directly in the code :D we have to be careful to remove those kind of comments when we close the issue